src/hooks/screen/useChatScreenHook.ts

```diff
@@ -153,15 +153,17 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
             channelList.description = message;
             channelList.lastUpdatedBy = userId;
             channelList.lastUpdatedAt = new Date().toISOString();
-            channelList.save(localDb);
+            await channelList.save(localDb);
           }
         }
 
         refresh('chat');
         refresh('channelList');
       }
 
-      const channelType = CHANNEL_TYPE_PERSONAL;
+      const channelType =
+        selectedChannel?.channelType === 'GROUP' ? CHANNEL_TYPE_GROUP : CHANNEL_TYPE_PERSONAL;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `channelType` is now set based on the `selectedChannel` object's `channelType` property. This allows for more dynamic handling of different channel types, improving the flexibility of the chat functionality.
```diff
-      const channelType = CHANNEL_TYPE_PERSONAL;
+      const channelType =
+        selectedChannel?.channelType === 'GROUP' ? CHANNEL_TYPE_GROUP : CHANNEL_TYPE_PERSONAL;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->